### DISCUSSION TITLE
@stack() fix still broken in certain scenarios

### DISCUSSION THREAD

**Discussion by BrekiTomasson** [livewire/livewire#4049] [7. Bugs (archived)] [answered]
On one of my websites, I have a code snippet that looks like this in my footer:

```
<script type="text/javascript">
  function defer() {
    if (window.$) {
      @stack('javascript')
    } else {
      setTimeout(function () {
        defer()
      }, 50)
    }
  }

  defer()

</script>
```

This allows me to `@push()` any javascript code that I want to run on pageload after ensuring that jQuery is available. However, with the most recent versions of LiveWire, this breaks, as the place where the `@stack()` command is placed receives  `<template livewire-stack="javascript"></template>`, which is not valid JavaScript code and breaks the entire script with `Uncaught SyntaxError: Unexpected token '<'`

Unfortunately, I can't think of a clean solution to this that isn't limited in other ways, but I trust you're a smarter man than I.

**Comment by Andreas-Halemba** [comment]
We are using twill.io and just installing livewire breaks the CMS as livewire changes the @stack behavior of blade templates.

**Comment by joshhanley** [comment]
@BrekiTomasson bugger! Yeah we didn't consider that use case. We incorrectly assumed people pushed their script tags into the stack.

So your use case where you actually wrap the `@stack` in a script tag will currently break, as we're using html `<template>` tags to denote the start and end of the stack block so Livewire can push to it.

I can't see a clean solution either, will have to think some more on it.

**Comment by joshhanley** [comment, marked as answer]
@BrekiTomasson and @Andreas-Halemba we've reverted the stacks feature and tagged 2.7.1.

**Reply by BrekiTomasson** [reply]
Thanks!
